scripts/coordinate.js

```diff
@@ -213,125 +213,121 @@ FriendlyChat.prototype.removeChats = function() {
+  var currentDoWhopID = userSessionCurrentDoWhop; // This is available from higher scope.
+  firebase.database().ref().child('DoWhopDescriptions/' + userSessionCurrentDoWhop).once('value').then(function(data) {
+    var doWhopSelector = document.getElementById('dowhop-selector-container');
+    var doWhopSelectorDiv = '';
+    var imageUrl =
+      (data.val().downloadURL && data.val().downloadURL.image1) ||
+      data.val().downloadURL ||
+      defaultDoWhopDescriptionImage;
+
+    var doWhopDescriptionTitle = data.val().titleDescription || 'Your DoWhops Will Appear Here';
+
+    doWhopSelectorDiv +=
+      "<section id='" +
+      data.key +
+      "' class='dowhop-selector-block''>" +
+      "<div class='dowhop-selector-header-top' style='background-image: url(" +
+      imageUrl +
+      ");'>" +
+      '<h1>' +
+      doWhopDescriptionTitle +
+      '</h1>' +
+      '</div>';
+
+    doWhopSelector.innerHTML = doWhopSelectorDiv; // Compose header block only.
+  });
+}
+
+function generateUserIcons(userObjectArray) {
+  var textOutput = '';
+  if (userObjectArray && userObjectArray.length > 0) {
+    _.map(userObjectArray, function(userObject) {
+      textOutput +=
+        '<div class="user-avatar-container">' +
+        "<a href='/profile.html?" +
+        userObject.uid +
+        "'>" +
+        "<div class='user-avatar'>" +
+        "<img class='user-avatar' src='" +
+        userObject.photoURL +
+        "'>" +
+        '</img>' +
+        '</div>' +
+        '<div class="user-handle">' +
+        userObject.displayName +
+        '</div>' +
+        '</a>' +
+        '</div>';
+    });
+  }
+  return textOutput;
+}
+
+function checkDoWhopDetails(userSessionCurrentDoWhop) {
+  var currentDoWhopID = userSessionCurrentDoWhop;
+  // Turning on a listener to the DoWhopDescriptions for a given DoWhop ID:
+  firebase.database().ref().child('DoWhopDescriptions/' + userSessionCurrentDoWhop).on('value', function(data) {
+    var doWhopSelectorBody = document.getElementById('dowhop-selector-container-body');
+    doWhopSelectorBody.innerHTML = ''; // Resetting the field to new info is up-to-date
+    var newOutPut = '';
+
+    var doWhopDescriptionTitle = data.val().titleDescription;
+    var renderWhyInformation = data.val().whyDescription;
+    var renderWhenInformation = data.val().whenDescription;
+    var renderWhereInformation = data.val().whereDescription;
+    var renderWhoDescription = data.val().whoDescription;
+    var renderWhoAmIInformation = data.val().whoAmIDescription || '';
+    var renderWhatInformation = data.val().whatDescription;
+    var renderWhenInformation = data.val().whenDescription;
+    var renderWhereInformation = data.val().whereDescription;
+
+    // Checking for updates to renderWhereInformation - DEV refactor.
+    if (data.val().whereAddress && data.val().whereAddress != 'By request') {
+      renderWhereInformation = data.val().whereAddress;
+    }
+
```

Comment:
Would this work without implicit coercion? Eg.
if (
      data.val().whenDateTime && data.val().whenDateTime !== ('By request' || 'tbd')
    ) {}